src/lib/isLatLong.js

```diff
@@ -7,5 +7,9 @@ export default function (str) {
   assertString(str);
   if (!str.includes(',')) return false;
   const pair = str.split(',');
+  if (
+    (pair[0].charAt(0) === '(' && pair[1].charAt(pair[1].length - 1) !== ')') ||
+    (pair[1].charAt(pair[1].length - 1) === ')' && pair[0].charAt(0) !== '(')
+  ) return false;
```

Comment:
This will fail if `str` has length of zero:

```
> const str = ""
> const pair = str.split(',')
> pair
[ '' ]
> pair[0].charAt(0)
''
> pair[1].charAt(pair[1].length - 1)
Thrown:
TypeError: Cannot read property 'charAt' of undefined
```

Reply:
@chriso, will it mean that we have to catch empty validations, for instance when a user passes an empty value for either latitude or longitude?

Reply:
Sorry, disregard my point. I see there's a `if (!str.includes(',')) return false;` guard above which disallows an empty string.

Reply:
The regex `^\(.*\)` might do the same validation, but I don't know if it will throw an error. 
I am open to possibilities/opinions on the same.